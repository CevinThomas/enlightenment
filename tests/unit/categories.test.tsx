import 'react-native';
import React from 'react';
import Categories from '../../src/views/categories';

import renderer from 'react-test-renderer';

test('Categories View Renders', () => {
    const tree = renderer.create(
        <Categories/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
