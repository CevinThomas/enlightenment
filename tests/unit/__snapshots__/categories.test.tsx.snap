// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories View Renders 1`] = `
<View
  style={
    Object {
      "height": 1200.6000000000001,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "column",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 0,
        "justifyContent": "center",
        "left": 0,
        "marginBottom": 20,
        "position": "absolute",
        "right": 0,
        "textAlign": "center",
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "overflow": "visible",
          "resizeMode": "contain",
          "width": 100,
        }
      }
    />
  </View>
</View>
`;
